app/services/store_sp_metadata_in_session.rb

```diff
@@ -82,12 +44,12 @@ def update_session
       request_url: sp_request.url,
       request_id: sp_request.uuid,
       requested_attributes: sp_request.requested_attributes,
-      ial: ial_value,
-      ial2: ial2_value,
-      ialmax: ialmax_value,
-      aal_level_requested: aal_level_requested_value,
-      piv_cac_requested: piv_cac_requested_value,
-      phishing_resistant_requested: phishing_resistant_value,
+      ial: parsed_vot&.ial_value_requested,
+      ial2: parsed_vot&.ial2_requested?,
+      ialmax: parsed_vot&.ialmax_requested?,
+      aal_level_requested: parsed_vot&.aal_level_requested,
```

Comment:
This commit makes a modification to the way these are set, but it does not change their usage. I don't see any changes in this pull request that removes the places we inspect `sp_session[:aal_level_requested]` and replaces them with `resolved_authn_context_result.aal2?` or `resolved_authn_context.phishing_resistant?`.

Reply:
I just looked an it looks like there are 2 places that the requested AAL is meaningful:

- [`ServiceProviderMfaPolicy`](https://github.com/18F/identity-idp/blob/e756d99fc620cbd59b6c35242f409429bf3b6912/app/policies/service_provider_mfa_policy.rb#L8): This may imply a bit of refactor here. We can probably entirely remove the requested AAL level and get a head start on [LG-12374](https://cm-jira.usa.gov/browse/LG-12374)
- [Remember device expiration interval](https://github.com/18F/identity-idp/blob/e756d99fc620cbd59b6c35242f409429bf3b6912/app/decorators/service_provider_session.rb#L91) in `ServiceProviderSession`